src/main/jbake/content/blog/2017/07/jaeger-with-security-proxy.adoc

```diff
@@ -0,0 +1,253 @@
+= Protecting Jaeger UI with a sidecar security proxy
+Juraci Paixão Kröhling
+2017-07-18
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger, opentracing
+
+In a production deployment of Jaeger, it may be advantageous to restrict access to Jaeger's Query
+service, which includes the UI. For instance, you might have internal security requirements to allow
+only certain groups to access trace data, or you might have deployed Jaeger into a public cloud.
+In a true microservices way, one possible approach is to add a sidecar to the Jaeger Query service,
+acting as a security proxy. Incoming requests hit our sidecar instead of reaching Jaeger's Query service
+directly and the sidecar would be responsible for enforcing the authentication and authorization constraints.
+
+ifndef::env-github[]
+image::/img/blog/2017/2017-07-18-teaser.png[Jaeger login screen]
+endif::[]
+ifdef::env-github[]
+image::../../../../../assets/img/blog/2017/2017-07-18-teaser.png[Jaeger login screen]
```

Comment:
One last tweak.  I'd move the bulleted list (lines 27 - 37) above the image and the explanation (lines 23 - 25).  I think it flows better if you get the explanation of what you're using for the demo before the image that shows how it all should fit together.